src/lib/tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -26,11 +26,23 @@ import {ViewportRuler} from '../../core/overlay/position/viewport-ruler';
   encapsulation: ViewEncapsulation.None,
 })
 export class MdTabNavBar {
+  _activeLinkChanged: boolean;
+  _activeLinkElement: ElementRef;
+
   @ViewChild(MdInkBar) _inkBar: MdInkBar;
 
-  /** Animates the ink bar to the position of the active link element. */
-  updateActiveLink(element: HTMLElement) {
-    this._inkBar.alignToElement(element);
+  /** Notifies the component that the active link has been changed. */
+  updateActiveLink(element: ElementRef) {
+    this._activeLinkChanged = this._activeLinkElement != element;
+    this._activeLinkElement = element;
+  }
+
+  /** Checks if the active link has been changed and, if so, will update the ink bar. */
+  ngAfterContentChecked(): void {
+    if (this._activeLinkChanged) {
```

Comment:
The initial set for `active` is performed while the component is setting up and before the content is ready, so the ink bar doesn't know what to align to. Subsequent calls would be fine but we're seeing an issue with the first load of the component.